exercises/practice/complex-numbers/complex_numbers_test.go

```diff
@@ -0,0 +1,127 @@
+package complex
+
+import (
+	"math"
+	"testing"
+)
+
+func TestNumber_Real(t *testing.T) {
+	for _, tt := range realTestCases {
+		t.Run(tt.description, func(t *testing.T) {
+			n := Number{tt.in.a, tt.in.b}
+			if got := n.Real(); got != tt.want {
+				t.Errorf("Number({%v, %v}).Real() = %v, want %v", tt.in.a, tt.in.b, got, tt.want)
+			}
+		})
+	}
+}
+
+func TestNumber_Imaginary(t *testing.T) {
+	for _, tt := range imaginaryTestCases {
+		t.Run(tt.description, func(t *testing.T) {
+			n := Number{tt.in.a, tt.in.b}
+			if got := n.Imaginary(); got != tt.want {
+				t.Errorf("Number({%v, %v}).Imaginary() = %v, want %v", tt.in.a, tt.in.b, got, tt.want)
+			}
+		})
+	}
+}
+
+func TestNumber_Add(t *testing.T) {
+	for _, tt := range addTestCases {
+		t.Run(tt.description, func(t *testing.T) {
+			n1 := Number{tt.n1.a, tt.n1.b}
+			n2 := Number{tt.n2.a, tt.n2.b}
+			if got := n1.Add(n2); got.Real() != tt.want.a || got.Imaginary() != tt.want.b {
```

Comment:
For the sake of consistency, for float point comparisons, we should take the "Interest is Interesting" exercise as reference:

https://github.com/exercism/go/blob/e9ec173379d1f6a156cee8ae050e4e99797051f8/exercises/concept/interest-is-interesting/interest_is_interesting_test.go#L8-L14

(feel free to copy paste the constant + function)